app/models/mixins/supports_feature_mixin.rb

```diff
@@ -125,6 +125,10 @@ def subclasses_supporting(feature)
       supported_subclasses.select { |subclass| subclass.supports?(feature) }
     end
 
+    def types_supporting(feature)
+      subclasses_supporting(feature).map(&:name)
+    end
```

Comment:
NOTE I needed this so that I could stub this in specs, since the e.g. ContainerNode objects in the specs is the base class I couldn't use `[ContainerNode].map(&:name)` because the actual STI type is `nil` and the `where(:type => ["ContainerNode"])` query returned nothing

Reply:
Yea, think we had this method before but we were able to avoid it for our previous use cases and removed it.

No surprise that we found a use case that needed it